app/test_main.py

```diff
@@ -1 +1,77 @@
-# write your code here
+from typing import Callable
+
+import pytest
+from unittest import mock
```

Comment:
Your code is great!
The only thing that needs to be fixed is the order of imports.
You correctly import `Callable` first, because it is a built-in module, but `unittest` is also built-in module, so it should also be placed after typing.
```suggestion
from typing import Callable
from unittest import mock

import pytest
```